src/data/markdown/docs/03 cloud/03 Integrations/05 Cloud APM.md

```diff
@@ -9,19 +9,35 @@ k6 Cloud platform supports exporting metrics to APM platforms, thereby enabling
 
 ## Supported APM Providers
 
-Each supported APM platform is called a provider in Cloud APM. As you'll see in each platform's respective section, the `provider` is a key passed to the APM configuration object and its value should match the providers listed below. Also, each provider has a separate set of configuration parameters. Therefore you need to visit your provider's page:
```

Comment:
`Each supported APM platform is called a provider in Cloud APM` sounded a bit confusing to me after we've changed the structure of this section, as some providers now have multiple platform pages.

I've also changed the word `provider` -> `platform` in a few cases, since we're actually linking to platform pages, not provider pages.

Reply:
@EdvinasDaugirdas I copied the current content. Feel free to change it. I'll do the same.